### PR TITLE
start using UIBackgroundConfiguration

### PR DESCRIPTION
**Pull request by harrieshin**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes
Following the best practices for https://developer.apple.com/videos/play/wwdc2020/10027

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1092)

### CODE REVIEW

File: ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift [hunk 1]
```diff
@@ -110,6 +110,10 @@ class DemoListViewController: DemoTableViewController {
         UserDefaults.standard.set(demo.title, forKey: DemoListViewController.lastDemoControllerKey)
     }
 
+    override func tableView(_ tableView: UITableView, canFocusRowAt indexPath: IndexPath) -> Bool {
```
Comment (sophialee0416) [inline ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:113]: I’m guessing this is for focus support? And we always want to be true right?

File: ios/FluentUI/Table View/TableViewCell.swift [hunk 2]
```diff
@@ -58,6 +58,25 @@ public enum TableViewCellAccessoryType: Int {
...
+// Different background color is shown based on if `TableViewCell` is shown in `TableView` as grouped or plain style
+@objc(MSFTableViewCellBackgroundStyleType)
+public enum TableViewCellBackgroundStyleType: Int {
+    case plain
+    case grouped
+    case clear
+
+    var defaultColor: UIColor {
+        switch self {
+        case .plain:
```
Comment (sophialee0416) [inline ios/FluentUI/Table View/TableViewCell.swift:70]: Should I replace these values with tokens later?

File: ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift (hunk 1 above)
Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:113]: yes

File: ios/FluentUI/Table View/TableViewCell.swift (hunk 2 above)
Comment (harrieshin) [reply ios/FluentUI/Table View/TableViewCell.swift:70]: i have a different draft

**Pull request closed without merging**